lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -630,4 +656,12 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
     ENV LIVEBOOK_TEAMS_FS "#{encrypt_file_systems_to_dockerfile(socket)}"
     """
   end
+
+  defp xla_target([]), do: nil
+
+  defp xla_target(XLA_TARGET: xla_target) do
```

Comment:
Let's change to map with `"XLA_TARGET" => ...`, since we can't match on arbitrary items in keyword list :)

Reply:
Ah actually we should just interpolate the whole keyword list as env vars, without checking for XLA_TARGET specifically :)